doc/doxygen/scummvm.doxyfile

```diff
@@ -0,0 +1,2484 @@
+# Doxyfile 1.8.14
```

Comment:
Why are there two doxyfiles?

Reply:
My bad, I overlooked this. I was experimenting with having two different doxyfiles with slightly different configurations (for example, for local builds and for the web server), but only one should have been committed here. I will remove.

This one (scummvm.doxyfile) is actually the corrent one.